src/libraries/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -16,7 +16,9 @@ public abstract class BaseGetSetTimes<T> : FileSystemTest
         private static string driveFormat = PlatformDetection.IsInAppContainer ? string.Empty : new DriveInfo(Path.GetTempPath()).DriveFormat;
 
         protected static bool isHFS => driveFormat != null && driveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase);
-        protected static bool isNotHFS => !isHFS;
+
+        protected static bool WithMillisecondResolution => PlatformDetection.IsBrowser || isHFS;
```

Comment:
If HFS and Browser do not support millisecond granularity, then aren't the names switched?

Reply:
Since this isn't limited to milliseconds (nanoseconds too), it might be better to name it that way. Not sure what a good name for that would be, maybe `WithoutPartialSeconds`/`WithPartialSeconds`  or maybe `LowTemporalResolution`/`HIghTemporalResolution`

Reply:
Going with `LowTemporalResolution/HighTemporalResolution`